govcd/vapp.go

```diff
@@ -830,7 +833,26 @@ func updateNetworkConfigurations(vapp *VApp, networkConfigurations []types.VAppN
 		types.MimeNetworkConfigSection, "error updating vApp Network: %s", networkConfig)
 }
 
-// Function RemoveAllNetworks unattach all networks from VAPP
+// RemoveAllNetworks detaches all networks from vApp
 func (vapp *VApp) RemoveAllNetworks() (Task, error) {
 	return updateNetworkConfigurations(vapp, []types.VAppNetworkConfiguration{})
 }
+
+// SetProductSectionList sets product section for a vApp
+//
+// The slice of properties "ProductSectionList.ProductSection.Property" is not necessarily ordered or returned as set before
+func (vapp *VApp) SetProductSectionList(productSection *types.ProductSectionList) (*types.ProductSectionList, error) {
```

Comment:
I would add a sentence - a hint - that this product section represents guest properties in all comments of the four public functions.

Reply:
Added

Reply:
Typo in "maniapulate", sounds funny though :D

Reply:
fixed